# descriptors of feature-points in a image
Descriptor[]  descriptor